AppCheckCore/Sources/Public/AppCheckCore/GACAppCheck.h

```diff
@@ -16,18 +16,43 @@
 
 #import <Foundation/Foundation.h>
 
-@import AppCheckCoreInterop;
-
 @class GACAppCheckToken;
 @protocol GACAppCheckProvider;
 @protocol GACAppCheckSettingsProtocol;
 @protocol GACAppCheckTokenDelegate;
+@protocol GACAppCheckTokenProtocol;
 
 NS_ASSUME_NONNULL_BEGIN
 
+NS_SWIFT_NAME(AppCheckCoreTokenHandler)
```

Comment:
typedefs should be unavailable for Swift like - https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h#L38 and should be avoided in public headers for ObjC if reasonable since they'll add tech debt to the API surface when we convert to Swift.

Reply:
Thanks for the heads up on the Swift interop! I think this one was reasonable to remove from the header.